packages/notification-center/src/components/notification-center/components/user-preference/channels.tsx

```diff
@@ -1,12 +1,12 @@
-import { Mail, Mobile, Sms } from '../../../../shared/icons';
+import { Bell, Direct, Mail, Mobile, Sms } from '../../../../shared/icons';
 import React from 'react';
 
 export const channels = [
   { type: 'sms', label: 'SMS', Icon: Sms, description: 'This is an sms' },
   { type: 'push', label: 'Push', Icon: Mobile, description: 'This is a push' },
   { type: 'email', label: 'Email', Icon: Mail, description: 'This is a mail' },
-  { type: 'in_app', label: 'In App', Icon: Mail, description: 'This is a mail' },
-  { type: 'direct', label: 'Direct', Icon: Mail, description: 'This is a mail' },
+  { type: 'in_app', label: 'In App', Icon: Bell, description: 'This is an in app' },
+  { type: 'direct', label: 'Direct', Icon: Direct, description: 'This is a direct' },
```

Comment:
I would consider removing the description all together @oba2311 not sure if it adds any value here, wdyt?